lib/webkit_inspection_protocol.dart

```diff
@@ -7,9 +7,12 @@
 library wip;
 
 import 'dart:async';
+import 'dart:collection';
 import 'dart:convert' show JSON, UTF8;
 import 'dart:io';
 
+part 'src/dom.dart';
```

Comment:
What about in-lining this into the `webkit_inspection_protocol.dart` file? Any strong preference on the use of parts?